lib/Wikia/src/Logger/WikiaLogger.php

```diff
@@ -212,11 +212,16 @@ public function setWebProcessor(WebProcessor $processor) {
 	 * @return Logger
 	 */
 	public function defaultLogger() {
-		return new Logger(
+		$logger = new Logger(
 			'default',
 			[$this->getSyslogHandler()],
 			[$this->getWebProcessor()]
 		);
+
+		// custom formatting of MediaWiki's Status class
+		$logger->pushProcessor( new StatusProcessor() );
```

Comment:
1. 
AFAIK this can be simplified to the form of:
```
$logger = new Logger(
  'default',
  [$this->getSyslogHandler()],
  [$this->getWebProcessor(), new StatusProcessor()]
);
```
2. all the other required objects got their own `getX()` methods that create them if necessary. I'd suggest to stick with this convention in this file